recipe/meta.yaml

```diff
@@ -1,13 +1,14 @@
-{% set name = "ucrt" %}
-{% set version = "10.0.22621.0" %}
+{% set version = "10.0.26100.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: ucrt
   version: {{ version }}
 
 source:
-  url: https://software-static.download.prss.microsoft.com/dbazure/888969d5-f34g-4e03-ac9d-1f9786c66749/22621.1.220506-1250.ni_release_WindowsSDK.iso
-  sha256: 0412fc77e364226c9395e3188a2d0ac5888317fecfba7bc2dbe003c2ddb5b3d3
+  # from https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/
+  url: https://go.microsoft.com/fwlink/?linkid=2272523
+  fn: release_WindowsSDK.iso
```

Comment:
I don't mind keeping the raw URL
```
>curl -ILSs https://go.microsoft.com/fwlink/?linkid=2272523 | grep "Location:"
Location: https://software-static.download.prss.microsoft.com/dbazure/888969d5-f34g-4e03-ac9d-1f9786c66749/26100.1.240331-1435.ge_release_WindowsSDK.iso
```
but the linkid is stable, and we're [using](https://github.com/conda-forge/clang-win-activation-feedstock/blob/39d5e62330c62067124a737da1dda238e532cc14/recipe/conda_build_config.yaml#L57-L58) this in clang-win-activation as well.

The `winsdk` bits from that feedstock (even though they're just activation) would presumably also make sense to be in this feedstock if we wanted; I tried digging through the history of this feedstock a bit; https://github.com/conda-forge/staged-recipes/pull/15376 started of as no-arch, but was changed to yes-arch in https://github.com/conda-forge/staged-recipes/pull/15377 - I don't really understand the benefit of not having noarch.